src/webgl/buffer.js

```diff
@@ -100,7 +100,14 @@ export default class Buffer extends Resource {
 
     if (!data) {
       type = type || GL.FLOAT;
-      bytes = bytes || 0;
+
+      // Workaround needed for Safari (#291):
+      // gl.bufferData with size (second argument) equal to 0 crashes.
+      // hence create zero sized array.
+      if (!bytes || bytes === 0) {
+        bytes = 0;
+        data = new Float32Array(0);
```

Comment:
> Maybe I should not change incoming 'data' object, but create new array and set it to this.data and leave data as is?

Well `data` is just a reference, setting it to a new `Array` doesn't change what was passed in.

The more important think is what we set `this.data` to. Should we still set it to `null`? It will be `null` if `data=null` and `bytes>0` so probably should be `null` if `bytes===0` as well?